lib/services/common.dart

```diff
@@ -4,9 +4,9 @@
 
 library dart_pad.common;
 
-final String serverURL = 'https://dart-services.appspot.com/';
-//final String serverURL = 'http://doc-test.dart-services.appspot.com/';
-//final String serverURL = 'http://127.0.0.1:8082/';
+//final String serverURL = 'https://dart-services.appspot.com/';
+final String serverURL = 'http://doc-test.dart-services.appspot.com/';
```

Comment:
Please revert this, and we'll need to do a server urpgrade before deploy the version.